packages/mobile/src/backup/BackupQuiz.tsx

```diff
@@ -124,6 +132,10 @@ export class BackupQuiz extends React.Component<Props, State> {
       mnemonicWords: mnemonicWordsUpdated,
       userChosenWords: userChosenWordsUpdated,
     })
+    CeloAnalytics.trackSubEvent(
+      CustomEventNames.backup_quiz_submit,
```

Comment:
not sure how `trackSubEvent` works but should we be tracking `backup_quiz_submit` on every backspace given we are using `startTracking` and `endTracking` to track the duration?

Reply:
i see now, first argument is what you want the second argument to be attached to